src/main/kotlin/uk/gov/justice/digital/hmpps/prisonregister/model/PrisonRepository.kt

```diff
@@ -18,7 +18,7 @@ interface PrisonRepository : JpaRepository<Prison, String>, JpaSpecificationExec
   fun findAllByPrisonIdIsIn(ids: List<String>): List<Prison>
 
   @Query(
-    "SELECT new uk.gov.justice.digital.hmpps.prisonregister.resource.dto.PrisonNameDto(p.prisonId, p.name) FROM Prison p WHERE (:active IS NULL OR p.active = :active) ORDER BY p.name",
+    "SELECT new uk.gov.justice.digital.hmpps.prisonregister.resource.dto.PrisonNameDto(p.prisonId, p.name) FROM Prison p WHERE (:active IS NULL OR p.active = :active) AND (:name IS NULL OR p.name = :name)ORDER BY p.name",
```

Comment:
This is confusing, isn't `name` the name of the prison?  e.g. HMP Moorland.  The `prisonId` would be `MDI` why is the parameter called `name`?

Reply:
I left 'name' cause the Resource is names to get the same information.
but i can refactor this to prisonid something I thought previously.